### PR TITLE
Bug 1142254 - close turn bluetooth on confirmation dialog while bluetooth state is off to on, r=arthur

### PR DESCRIPTION
**Pull request by ian-liu**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[ian-liu Ian Liu (ian-liu)](https://github.com/ian-liu) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=1fec5f9a24b7)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[ian-liu Ian Liu (ian-liu)](https://github.com/ian-liu) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=4a888aeb5b5e)

File: apps/bluetooth/js/transfer.js [hunk 1]
```diff
@@ -67,6 +74,10 @@ navigator.mozL10n.once(function showPanel() {
           debug(msg);
           confirmTurnBluetoothOn();
         }
+      } else {
+        if (!dialogConfirmBluetooth.hidden) {
```
Comment (crh0716) [inline apps/bluetooth/js/transfer.js]: nit: I think we can set it to true without the check.

**Comment by try-server-hook** [comment]
[ian-liu Ian Liu (ian-liu)](https://github.com/ian-liu) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=c9a6a7d7bf7f)

**Comment by ian-liu** [comment]
All tests are passed on try-server. We can merge the pr.

**Pull request merged**